test/test/hits-unchanged.js

```diff
@@ -0,0 +1,74 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const expect = chai.expect;
+const should = chai.should();
+chai.use(chaiHttp);
+
+const constants = require('./constants');
+const util = require('./util');
+const expectUnchanged = require('./compare-responses').expectUnchanged;
+
+// Test that a hits search for a pattern returns the correct number of hits and docs,
+// and optionally test that the first hit matches (either JSON or text).
+function expectHitsUnchanged(testName, params) {
```

Comment:
`testName` is also the test file name right?

Reply:
Correct, added to comments.